x-pack/plugins/apm/public/components/app/correlations/correlations_chart.tsx

```diff
@@ -160,82 +164,95 @@ export function CorrelationsChart({
-                    fieldValue: value,
-                  },
-                }
-              )}
-              xScaleType={ScaleType.Log}
-              yScaleType={ScaleType.Log}
-              data={histogram}
-              curve={CurveType.CURVE_STEP_AFTER}
-              xAccessor="key"
-              yAccessors={['doc_count']}
-              color={euiTheme.eui.euiColorVis2}
-            />
-          )}
-      </Chart>
-      <EuiSpacer size="s" />
+        <Chart>
+          <Settings
+            rotation={0}
+            theme={chartTheme}
+            showLegend
+            legendPosition={Position.Bottom}
+          />
+          <LineAnnotation
+            id="annotation_1"
+            domainType={AnnotationDomainType.XDomain}
+            dataValues={annotationsDataValues}
+            style={annotationsStyle}
+            marker={`${markerPercentile}p`}
+            markerPosition={'top'}
+          />
+          <Axis
+            id="x-axis"
+            title=""
+            position={Position.Bottom}
+            tickFormat={(d) => {
+              const unit = getDurationUnitKey(d, 1);
+              const converted = getUnitLabelAndConvertedValue(unit, d);
+              const convertedValueParts = converted.convertedValue.split('.');
+              const convertedValue =
+                convertedValueParts.length === 2 &&
+                convertedValueParts[1] === '0'
+                  ? convertedValueParts[0]
+                  : converted.convertedValue;
+              return `${convertedValue}${converted.unitLabel}`;
+            }}
+          />
+          <Axis
+            id="y-axis"
+            domain={yAxisDomain}
+            title={i18n.translate(
+              'xpack.apm.correlations.latency.chart.numberOfTransactionsLabel',
+              { defaultMessage: '# transactions' }
+            )}
+            position={Position.Left}
+            ticks={yTicks}
+          />
+          <AreaSeries
+            id={i18n.translate(
+              'xpack.apm.correlations.latency.chart.overallLatencyDistributionLabel',
+              { defaultMessage: 'Overall latency distribution' }
+            )}
+            xScaleType={ScaleType.Log}
+            yScaleType={ScaleType.Log}
+            data={overallHistogram ?? []}
+            curve={CurveType.CURVE_STEP_AFTER}
+            xAccessor="key"
+            yAccessors={['doc_count']}
+            color={euiTheme.eui.euiColorVis1}
+            fit="lookahead"
+          />
+          {Array.isArray(histogram) &&
+            field !== undefined &&
+            value !== undefined && (
+              <AreaSeries
+                id={i18n.translate(
+                  'xpack.apm.correlations.latency.chart.selectedTermLatencyDistributionLabel',
+                  {
+                    defaultMessage: '{fieldName}:{fieldValue}',
+                    values: {
+                      fieldName: field,
+                      fieldValue: value,
+                    },
```

Comment:
Any reason to change the names? Does it improve readability?

```suggestion
                    defaultMessage: '{field}:{value}',
                    values: { field, value },
```

Reply:
Good catch, it was copy pasted like this from another chart. Looking at it again, I don't think we need translation here at all since it's just values and no static text. Fixed in 07780b9.